eth/beacon/types/validator_records.py

```diff
@@ -19,36 +19,44 @@ class ValidatorRecord(rlp.Serializable):
     Note: using RLP until we have standardized serialization format.
     """
     fields = [
-        # The validator's public key
-        ('pubkey', int256),
-        # What shard the validator's balance will be sent to after withdrawal
-        ('withdrawal_shard', int16),
-        # And what address
+        # BLS public key
+        ('pubkey', uint256),
+        # Withdrawal shard number
```

Comment:
`withdrawal_shard` and `withdrawal_address` were replaced with `withdrawal_credentials: 'hash32'`